client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice-dialog.jsx

```diff
@@ -165,11 +166,11 @@ function PendingGSuiteTosNoticeDialog( props ) {
 					<Button
 						href={ getLoginUrlWithTOSRedirect( props.user, props.domainName ) }
 						onClick={ onResetPasswordLogInClick }
-						primary={ isCopied }
+						primary
```

Comment:
We can make this button always primary as we shouldn't assume that the user will use the `Copy` button.